src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -492,4 +493,24 @@ protected function createChildContext(array $parentContext, $attribute/*, string
 
         return $parentContext;
     }
+
+    protected function attributeAllowedWithVersion(array $context, AttributeMetadataInterface $attributeMetadata): bool
+    {
+        $sinceVersion = $attributeMetadata->getSince();
+        $untilVersion = $attributeMetadata->getUntil();
+
+        if (!isset($context['version'])) {
+            return true;
+        }
+
+        if (null !== $sinceVersion && version_compare($sinceVersion, $context['version'], '>')) {
```

Comment:
I guess it should be '>=' here such as the first example here: https://www.php.net/manual/fr/function.version-compare.php
I would like to use Since like this: '@Since("2.0")' to say 2.0 included.
I would like to use Until like this '@Until("2.0") to say 2.0 excluded.
Thanks a lot.